src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyResultSetCursorTest.java

```diff
@@ -143,7 +143,7 @@ public void testMultiplePreparedStatementAndResultSet() throws SQLException {
                     verifyDestinationTableData(expectedBigDecimals.length * 3);
                 }
 
-                String sql = "insert into " + AbstractSQLGenerator.escapeIdentifier(desTable) + " values (?,?,?,?)";
+                String sql = "insert into " + AbstractSQLGenerator.escapeIdentifier(desTable) + "(c1,c2,c3,c4) values (?,?,?,?)";
```

Comment:
Adding columns here is not required if the first column is Identity. Identity column cannot be inserted, hence always excluded.